backend/src/server.ts

```
@@ -169,6 +169,56 @@ export async function createApp() {
+  app.use('/functions/:slug', (req: Request, res: Response, next: NextFunction) => {
+    if (req.method === 'GET' || req.method === 'HEAD') {
+      req.rawBody = Buffer.alloc(0);
+      next();
+      return;
+    }
+    const chunks: Buffer[] = [];
+    let done = false;
+    let totalBytes = 0;
+
+    const onData = (chunk: Buffer) => {
+      if(done) return;
```

ESLint

[error] Insert `·`

(prettier/prettier)